src/home_audit.ts

```diff
@@ -542,13 +542,21 @@ function getAdditionalFloorZoneValidations(floors: Floor[], roofs: Roof[], about
 /**
  * Iterates over an array of objects, calculating the sum of a given field name from each object
  */
-function getSumOfObjectPropertiesByFieldName(objects:{[key: string]: any}[], field_name:string):number
+function getDecimalSumOfObjectPropertiesByFieldName(objects:{[key: string]: any}[], field_name:string):number
```

Comment:
I like the idea of taking the Math.floor() out of the method that sums the field values. I would go further and not put it in getSumOfObjectProperisByFieldName() either – I think we really want to handle any tolerances where the sum is used.

Reply:
Those are used for roof & floor areas